src/Misc/interfaces/iPerfViewer.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Defines what data is needed to graph a point on the graph.
+ */
+export interface IPerfPoint {
+    /**
+     * The x coordinate of the point.
+     */
+    x: number;
+
+    /**
+     * The y coordinate of the point.
+     */
+    y: number;
+}
+
+/**
+ * Defines the shape of a dataset that our graphing service uses for drawing purposes.
+ */
+export interface IPerfDataset {

```

Comment:
I assume that most (if not all) datasets will be time-based. This can be added later, of course, but shouldn't we have some form of a timestamp here?

Or - is the x axis of the graph anything other than time?

Reply:
Yes! We have the same idea the x axis will be plotting time! I believe the timestamp can be represented as a number. 

Would you want me to rename x: number to timestamp: number to make this more clear? Or is there a potential performance concern that you can see with storing timestamps as numbers?